src/orquestra/sdk/_base/_api/_wf_run.py

```diff
@@ -138,6 +138,7 @@ def start_from_ir(
         config: t.Union[RuntimeConfig, str],
         workspace_id: t.Optional[WorkspaceId] = None,
         project_id: t.Optional[ProjectId] = None,
+        project_dir: t.Optional[t.Union[str, Path]] = None,
```

Comment:
**question**: what's the reason for `project_dir`? We used this in the past, but it's a *distant past* and I don't know any use case for anymore. Let's avoid cargo culting if we can.

Reply:
The only reason Ive added project_dir to this function, is so the code of this function can be reused in the WorkflowDef.run() without worrying about any backward incompatibility.
I do agree project_dir is a concept of a past - we might want to add a ticket to investigate if we can remove it, but here it is added just to reduce code duplication between WorkflowDef.run and WorkflowRun.start_from_ir()